style.css

```diff
@@ -253,24 +263,31 @@ h2{
     align-items: center;
 }
 
+.banner-titulo img {
+    width: 40%;
+}
+
 .banner-dois{
     background-image: url("img/sekiro_banner.jpg");
-    background-size: contain;
-    width: 627px;
+    background-size: cover;
+    width: calc(50% - 10px);
     height: 429px;
     display: flex;
     align-items: flex-end;
 }
 
+.produtos {
+    display: flex;
+    align-items: center;
+    margin: 80px auto;
+}
+
 .produtos img{
-    position: relative;
-    left: 324px;
-   
-    bottom: 37px
+    width: 24px;
+    margin-right: 24px;
```

Comment:
Aqui apenas removi o posicionamento forçado com left e inclui ele para estar antes do texto no html. Então se você observar, apenas adiciono uma margin-right para que o texto fique afastado do icone, e em cima, na classe `.produtos` uso um align-items: center. Assim a imagem fica correta e o texto também, sem necessidade de adicionar a margin nele

![image](https://user-images.githubusercontent.com/3194874/121446257-7c5ab300-c969-11eb-95e0-950b010a053b.png)